extensions/common/xwalk_external_extension.h

```diff
@@ -53,7 +54,7 @@ class XWalkExternalExtension : public XWalkExtension {
 
   // Variables from the browser process. Usually things like currently-running
   // application ID.
-  base::ValueMap runtime_variables_;
+  base::DictionaryValue::Storage runtime_variables_;
```

Comment:
https://codereview.chromium.org/2000803003/diff/160001/base/values.h lines 46 and 210